src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -39,12 +39,16 @@ internal AcquireTokenForClientParameterBuilder(IConfidentialClientApplicationExe
         }
 
         /// <summary>
-        /// Specifies if the token request will ignore the access token in the application token cache
-        /// and will attempt to acquire a new access token using client credentials.
+        /// Specifies if the client application should ignore access tokens when reading the token cache. 
```

Comment:
```suggestion
        /// Specifies if the client application should ignore access tokens already present in the token cache and attempt to acquire a new one. 
```
nit: I think it ready better and is a little more clear.